og.module

```diff
@@ -159,14 +159,10 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
     return $access;
   }
 
-  if ($entity_type_id == 'node') {
-    $node_access_strict = \Drupal::config('og.settings')->get('node_access_strict');
+  $entity_access_strict = \Drupal::config('og.settings')->get('entity_access_strict');
 
-    // Otherwise, ignore or deny based on whether strict node access is set.
-    return AccessResult::forbiddenIf($node_access_strict);
-  }
-
-  return AccessResult::forbidden();
```

Comment:
This makes sense to me, I don't see the point of special casing nodes, and providing a blanket ban on any other entity types.